apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -77,22 +56,112 @@ import { HashLinkPageProps } from "../../../pages/d/[link]/book";
 import { TeamBookingPageProps } from "../../../pages/team/[slug]/book";
 
 type BookingPageProps = BookPageProps | TeamBookingPageProps | HashLinkPageProps;
+const BookingFields = ({
+  fields,
+  locations,
+  rescheduleUid,
+  isDynamicGroupBooking,
+}: {
+  fields: BookingPageProps["eventType"]["bookingFields"];
+  locations: LocationObject[];
+  rescheduleUid?: string;
+  isDynamicGroupBooking: boolean;
+}) => {
+  const { t } = useLocale();
+  const { watch, setValue } = useFormContext();
+  const locationResponse = watch("responses.location");
 
-type BookingFormValues = {
-  name: string;
-  email: string;
-  notes?: string;
-  locationType?: EventLocationType["type"];
-  guests?: { email: string }[];
-  address?: string;
-  attendeeAddress?: string;
-  phone?: string;
-  hostPhoneNumber?: string; // Maybe come up with a better way to name this to distingish between two types of phone numbers
-  customInputs?: {
-    [key: string]: string | boolean;
-  };
-  rescheduleReason?: string;
-  smsReminderNumber?: string;
+  return (
+    // TODO: It might make sense to extract this logic into BookingFields config, that would allow to quickly configure system fields and their editability in fresh booking and reschedule booking view
+    <div>
+      {fields.map((field, index) => {
+        // During reschedule by default all system fields are readOnly. Make them editable on case by case basis.
+        // Allowing a system field to be edited might require sending emails to attendees, so we need to be careful
+        let readOnly =
+          (field.editable === "system" || field.editable === "system-but-optional") && !!rescheduleUid;
+        let noLabel = false;
+        let hidden = false;
```

Comment:
Why not?

```suggestion
        let hidden = !!field.hidden;
```

Reply:
BTW this solved this https://github.com/calcom/cal.com/pull/6560#issuecomment-1432026165

Reply:
Yeah it should be that way only 🙏 It's a recent regression while fixing another bug